apps/wallet/src/ui/app/shared/faucet/FaucetRequestButton.tsx

```diff
@@ -46,9 +45,6 @@ function FaucetRequestButton({
                         <FaucetMessageInfo error={error.message} />
                     ),
                 });
-                trackEvent('RequestGas', {
-                    props: { source: trackEventSource, networkName },
```

Comment:
You can probably get rid of trackSource, and maybe network name?

Reply:
yep sorry just threw this up during my meeting lol